### DISCUSSION TITLE
Casey Thread

### DISCUSSION THREAD

**Discussion by GaloisField2718** [ordinals/ord#3385] [Q&A]
Hey, do you have any understanding of the thread made yesterday by @casey ? 

https://x.com/rodarmor/status/1772739766657692022

I tried to read the code for runes into `ord` and follow each updates. But I didn't get anything from the Casey's tweets. 
Maybe some discussions here can help to clarify it.

**Comment by cryptoni9n** [comment]
Hi GaloisField2718 - it doesn't look like this discussion got much traction.  Do you still have anything specific that you'd like to discuss about runes?

**Reply by GaloisField2718** [reply]
Hi Cryptoni9ne, 
Not specifically, just I still don't understand how to properly uncipher a runestone from hex of the tx "by hands".
